modules/nw-exposing-service.adoc

```diff
@@ -85,7 +85,7 @@ ifdef::nodeport[]
 +
 [source,terminal]
 ----
-$ oc expose dc mysql-80-rhel7 --type=NodePort --name=mysql-ingress
+$ oc expose svc/mysql-80-rhel7 --type=NodePort --name=mysql-nodeport --generator="service/v2"
 ----
```

Comment:
@nalhadef Just to confirm, Is this part of our PR ?

Reply:
Thanks, @pdhamdhe. I've removed the erroneous entries.